api/integration/install_test.go

```diff
@@ -162,6 +162,7 @@ func TestConfigureInstallation(t *testing.T) {
 			installController, err := install.NewInstallController(
 				install.WithHostUtils(tc.mockHostUtils),
 				install.WithRuntimeConfig(rc),
+				install.WithEnvSetter(&testEnvSetter{}),
```

Comment:
Wondering if we're missing a set of assertions here that validate:
- The environment variables are set
- All these runtime config fields are updated accordingly - https://github.com/replicatedhq/embedded-cluster/blob/4672f1caddd8398c3506b1e66bcee79a0b8b7c42/api/controllers/install/installation.go#L65-L69